src/components/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { SiProbot } from 'react-icons/si';
+import '../../App.css';
+
+const Header = () => {
```

Comment:
Nice reusable component, you can edit the code in one place and it will update on all pages!